<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGuiSetupSensor</class>
 <widget class="QWidget" name="QtGuiSetupSensor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtGuiSetupSensor</string>
  </property>
  <widget class="QTabWidget" name="setupSensorParams">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>394</width>
     <height>630</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>700</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_brightless">
    <attribute name="title">
     <string>Brightless</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="GB_exprosure">
       <property name="title">
        <string>Exposure Auto</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="PB_continuous">
          <property name="text">
           <string>Continuous</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="PB_once">
          <property name="text">
           <string>Once</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="PB_off">
          <property name="text">
           <string>Off</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_roi">
    <attribute name="title">
     <string>ROI</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Image Format</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Pixel Format:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="3">
         <widget class="QComboBox" name="ComBox_pixelFormat">
          <item>
           <property name="text">
            <string>Mono8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mono12</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mono12Packed</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Binning Horizontal:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QSpinBox" name="SpinB_binningHor">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>8</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Binning Vertical:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QSpinBox" name="SpinB_binningVer">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>11</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>67</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>ROI</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="Widget" name="setRoiWid" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>256</width>
              <height>256</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>256</width>
              <height>256</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>ROI coordinate</string>
          </property>
          <layout class="QGridLayout" name="gridLayout" columnstretch="2,1,2,1">
           <item row="1" column="1">
            <widget class="QSpinBox" name="SpinB_height">
             <property name="maximum">
              <number>3248</number>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>OffsetY</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Height</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="SpinB_ofsetY">
             <property name="maximum">
              <number>3248</number>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QSpinBox" name="SpinB_ofsetX">
             <property name="maximum">
              <number>4872</number>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Width</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>OffsetX</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QSpinBox" name="SpinB_width">
             <property name="maximum">
              <number>4872</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="PB_full">
            <property name="text">
             <string>Full</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="PB_oneQuarter">
            <property name="text">
             <string>1/4</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="PB_oneEighth">
            <property name="text">
             <string>1/8</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,1">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="PB_setRoi">
            <property name="text">
             <string>Set ROI</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_delay">
    <attribute name="title">
     <string>Delay</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,1,2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Trigger interval</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinB_trigerDelay">
         <property name="maximum">
          <number>10000</number>
         </property>
         <property name="value">
          <number>23</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>557</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Widget</class>
   <extends>QWidget</extends>
   <header>widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
